koku/api/tags/azure/openshift/queries.py

```diff
@@ -26,7 +26,7 @@ class OCPAzureTagQueryHandler(AzureTagQueryHandler, OCPTagQueryHandler):
     """Handles tag queries and responses for OCP-on-Azure."""
 
     data_sources = [{'db_table': OCPAzureCostLineItemDailySummary, 'db_column': 'tags'}]
-    provider = 'OCP_AZURE'
+    provider = 'OCP_Azure'
```

Comment:
Maybe have a constant for this in the Providers class, as it is used more than once across files?